app/src/androidTest/kotlin/at/bitfire/davdroid/webdav/DavDocumentsProviderTest.kt

```diff
@@ -101,9 +101,9 @@ class DavDocumentsProviderTest {
 
         // Assert new children were inserted into db
         assertEquals(3, db.webDavDocumentDao().getChildren(parent.id).size)
-        assertEquals("Secret_Document.pages", db.webDavDocumentDao().getChildren(parent.id)[0].displayName)
+        assertEquals("Library", db.webDavDocumentDao().getChildren(parent.id)[0].displayName)
```

Comment:
Should we update the test name?

Reply:
Hm what do you mean? I think we just had to switch the order because the entries are now sorted by default.